Assets/Prefabs/XR Origin (Player).prefab

```diff
@@ -1199,8 +1199,17 @@ LineRenderer:
         outSlope: 0
         tangentMode: 0
         weightedMode: 0
-        inWeight: 0.33333334
-        outWeight: 0.33333334
+        inWeight: 0
```

Comment:
Same with these ones below?

Reply:
Good catch. These changes shouldn't be there! Reverted in the new commit